### PR TITLE
Add com.krioltech.fnmap

### PR DESCRIPTION
**Pull request by gmckesey**
### Please confirm your submission meets all the criteria

- [ X] Please describe your application briefly.
- [ X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ X] My pull request follows the instructions at [App Submission][submission].
- [ X] I have [built][build] and tested the submission locally.
- [ X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by bbhtt** [comment]
why did you close https://github.com/flathub/flathub/pull/5202 ? It just needed a rebase.

**Comment by bbhtt** [comment]
bot, build com.krioltech.fnmap

File: com.krioltech.fnmap.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - buildsystem: simple
+    name: fnmap
+    build-commands:
+      - cp -R fnmap-archive/bin/bundle /app/fnmap
+      - chmod +x /app/fnmap/fnmap
+      - mkdir -p /app/bin/
+      - ln -s /app/fnmap/fnmap /app/bin/fnmap
+      - install -Dm644 fnmap-archive/com.krioltech.fnmap.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 fnmap-archive/fnmap.desktop /app/share/applications/fnmap.desktop
+      - install -Dm644 fnmap-archive/icons/fnmap.png /app/share/icons/com.krioltech.fnmap.png
```
Comment (bbhtt) [inline com.krioltech.fnmap.yml:32]: What size is this? This is the wrong location https://docs.flatpak.org/en/latest/conventions.html#application-icons

Comment (gmckesey) [reply com.krioltech.fnmap.yml]: Icon is now installed in the correct location.

File: com.krioltech.fnmap.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=xdg-documents
```
Comment (bbhtt) [inline com.krioltech.fnmap.yml:14]: Why is home needed now? xdg-documents with home is redundant

Comment (gmckesey) [reply com.krioltech.fnmap.yml]: removed home privilege.

**Comment by gmckesey** [comment]
@bbhtt are you waiting for any more changes?

**Comment by bbhtt** [comment]
Nope, missed the last notification.

bot, build com.krioltech.fnmap

File: com.krioltech.fnmap.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-documents
```
Comment (bbhtt) [inline com.krioltech.fnmap.yml:13]: why do you need xdg-documents? The filepicker seems to support filechooser portal.

image

Comment (gmckesey) [reply com.krioltech.fnmap.yml]: I have removed the xdg-documents privilege.

**Comment by bbhtt** [comment]
bot, build com.krioltech.fnmap

**Pull request closed without merging**